ee-9/feature-pack/pom.xml

```diff
@@ -779,6 +779,38 @@
                     <groupId>com.sun.istack</groupId>
                     <artifactId>istack-commons-tools</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.ironjacamar</groupId>
+                    <artifactId>ironjacamar-common-api</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.ironjacamar</groupId>
+                    <artifactId>ironjacamar-common-impl-jakarta</artifactId>
+                </exclusion>
```

Comment:
Hi @tadamski , you have to exclude the old non -jakarta versions from this feature pack.

Reply:
hi @yersan, fixed